src/include/uapi/ipc/header.h

```diff
@@ -103,6 +103,7 @@
 #define SOF_IPC_PM_CLK_GET			SOF_CMD_TYPE(0x005)
 #define SOF_IPC_PM_CLK_REQ			SOF_CMD_TYPE(0x006)
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
+#define SOF_IPC_PM_STATE_SET			SOF_CMD_TYPE(0x008)
```

Comment:
Should we also have a get state ?

Reply:
We may have (I have added the command to have a symmetry with set and as a provision for the future). It's worth to keep in mind that FW will not return value while in D3 and requesting IPC response while FW is in D0ix (unless its repeated IPC_PM_STATE_SET(D0ix)) will change FW mode of work back to D0(i0).